Automattic/simplenote-ios
# Search Minimap

### Details
This PR adds a touchable minimap to the Note when in search mode. 

Closes #953 

@jleandroperez Can I bug you with this one? Thanks!
cc @SylvesterWilmott 

### Test
1. Have both big and small notes on hand
2. Search for something
3. Open a note

- [x] Search minimap is shown on the right side
- [x] Tapping on a bar would scroll to the result and highlight it
- [x] Dragging finger on the minimap snaps moves to the result close to the finger
- [x] Dragging finger on the minimap has haptic feedback on supported devices
- [x] Dismissing search dismisses minimap
- [ ] Starting editing a note dismisses minimap

### Release
`RELEASE-NOTES.txt` was updated in https://github.com/Automattic/simplenote-ios/pull/1004/commits/a95205cf8c781900c36a11fbd1e248b8b478e6fb with:
 > Added Search Minimap to quickly navigate search hits #953

<Img src="https://user-images.githubusercontent.com/54751/98992251-479a9080-252d-11eb-9060-01a7b2f4d3cd.PNG" width="300" />
